master/cloud-common.md

```diff
@@ -77,3 +79,4 @@ After you set up the Ark server, try these examples:
 [3]: namespace.md
 [4]: ibm-config.md
 [19]: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming
+[20]: https://github.com/heptio/ark/blob/master/docs/restic.md
```

Comment:
This is my bad, but this link should be changed (can do it in a separate PR on master)